### PR TITLE
Removed duplicates from book_authors table, corrected readme accordin…

### PR DESCRIPTION
**Pull request by RogeredBacon**
…gly and added solutions for the "hard" challenges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,8 +277,20 @@ Doing these is not required! Only look at these if you have time at the end.
...
 
+##### Expected result
+
+Ted Burns
+
 * Which location has the higher figure for books per author?
 
+##### Expected result
+
+(Undefined), Nazareth
```
Comment (mattlub) [inline README.md]: is this supposed to be just "Nazareth"?

**Comment by mattlub** [comment]
@RogeredBacon could you resolve the conflicts?

Did you replace the duplicates from `book_authors` table, instead of removing them, so that the expected results didn't change?

**Comment by mattlub** [comment]
@eliascodes @m4v15 @shiryz @arrested-developer 

I picked up this old PR just to fix the merge conflict. It just adds solutions for the hard challenges. It also fixes the initial `data.sql` to remove duplicated entries in one of the tables- this had caused an error when trying to add a primary key for the table in one of the hard challenges.

I've checked that the solutions/expected results are still correct. :shipit:

**Comment by m4v15** [comment]
I ran this and it works for me, @eliascodes if you're happy I think we can probably just go ahead and merge this as it doesn't change anything about the workshop?

**Review by m4v15 — APPROVED** [review]

**Review by eliasmalik — APPROVED** [review]

**Pull request merged**